packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -12,15 +12,20 @@ import * as React from 'react'
 import styled from 'styled-components'
 import { Tick, TickLarge } from '../icons'
 
-const Wrapper = styled.li`
-  width: 100%;
-  height: 40px;
+const Label = styled.label<{ disabled?: boolean }>`
+  list-style-type: none;
```

Comment:
list-style-type is not needed anymore as it's a label?

Reply:
this is redundant here as @naftis mentioned, @jpye-finch let's remove this from here and from `RadioButton` too